css/selectors/grammar-error.json

```
@@ -23,7 +23,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
+              "version_added": "preview",
               "impl_url": "https://webkit.org/b/175784"
```

Should we remove the bug reference? It seems like that bug is stalled.
https://bugs.webkit.org/show_bug.cgi?id=265997 and https://bugs.webkit.org/show_bug.cgi?id=265879 are more current and are fixed, so I think we need to bug reference anymore here. What do you think?

```suggestion
              "version_added": "preview"
```

---

I don't think we need them since they are fixed bugs.